kernel/kernel-util/src/main/java/org/sakaiproject/util/EmailNotification.java

```diff
@@ -645,6 +645,24 @@ protected int getOption(User user, String notificationId, String resourceFilter,
 		String type = getType(resourceFilter);
 		if (type != null)
 		{
+            // First, check the overrides.
```

Comment:
If I may interject with my personal preference/opinion:

Spaces are MUCH better than tabs. Simply because one space == one column. Therefore, no matter what editor you use to open a file, the indentation will ALWAYS be the same. With tabs, it is up to the settings of that particular editor.

And no, there's definitely no "official" policy on tabs vs. spaces. Just look through the code base and you'll see this. Personally, I believe that as long as the code is sound and works as intended, the formatting should play a very distant second fiddle... :)